modules/packages/UnorderedAtomics.chpl

```diff
@@ -162,15 +162,15 @@ module UnorderedAtomics {
   }
 
   /*
-     Fence any pending unordered atomics. Note that this is a global fence
-     across all locales.
+     .. warning::
+       This function has been deprecated - please use
+       :proc:`unorderedAtomicTaskFence()` instead. Note that this function has
+       been deprecated without a full release of support because the previous
+       global fence semantics imposed expensive implementation requirements is
+       not meant to to be the main use case.
    */
   inline proc unorderedAtomicFence(): void {
-    if CHPL_NETWORK_ATOMICS != "none" {
-      extern proc chpl_comm_atomic_unordered_fence();
-      coforall loc in Locales do on loc {
-        chpl_comm_atomic_unordered_fence();
-      }
-    }
+    compilerError("unorderedAtomicFence() is no longer supported - please use unorderedAtomicTaskFence() instead");
+    unorderedAtomicTaskFence();
```

Comment:
With the `compilerError()` here, do we actually need the body contents?  (If there's a way to turn off `compilerError()`, maybe so.)

Reply:
A sufficiently motivated user could compile with `--ignore-errors` and I wanted to be a little bit friendly since we're removing something instead of deprecating.